test/e2e/create_cluster_test.go

```diff
@@ -1396,6 +1432,77 @@ func TestCreateClusterPrivateWithRouteKAS(t *testing.T) {
 	testCreateClusterPrivate(t, true)
 }
 
+// testCreateClusterWithDisabledCapabilities implements a test that creates a cluster
+// with the ImageRegistry capability disabled, then attempts to enable it.
+func testCreateClusterWithDisabledCapabilities(ctx context.Context, t *testing.T, g Gomega, mgtClient crclient.Client, hostedCluster *hyperv1.HostedCluster) {
+	if os.Getenv("TECH_PREVIEW_NO_UPGRADE") != "true" {
```

Comment:
isn't this PR removing the gate?

Reply:
Oops, I must've messed one of my interactive rebases. Fixed.